src/Docfx.Build/XRefMaps/XRefArchiveBuilder.cs

```diff
@@ -36,88 +36,32 @@ public async Task<bool> DownloadAsync(Uri uri, string outputFile)
         }
     }
 
-    private async Task<string> DownloadCoreAsync(Uri uri, XRefArchive xa, bool isMajor)
+    private async Task<string> DownloadCoreAsync(Uri uri, XRefArchive xa)
     {
         IXRefContainer container;
         container = await _downloader.DownloadAsync(uri);
         if (container is not XRefMap map)
         {
-            // not support download an xref archive, or reference to an xref archive
+            // XRefArchive is not supported by `docfx download`.
+            Logger.LogWarning($"Download an xref archive, or reference to an xref archive is not supported. URI: {uri}");
             return null;
         }
-        if (map.Redirections?.Count > 0)
-        {
-            await RewriteRedirections(uri, xa, map);
-        }
-        if (map.References?.Count > 0 && map.HrefUpdated != true)
-        {
-            if (string.IsNullOrEmpty(map.BaseUrl))
```

Comment:
Does it make sense that URLs are relative to the `baseUrl` in xrefmap if exist, otherwise relative to the download location?

Reply:
I've added following logics for xrefmaps that don't define `baseUrl` and using relative urls. (https://github.com/dotnet/docfx/pull/9721/commits/6e665d3d0d590fe9c52aa09bcb16909c978dd84e)
1. If `baseUrl` is not defined. Set `download location` as `baseUrl`.
2. If `baseUrl` is not defined. Try to resolve href that are using relative URL to absolute URL.

In addition. I've added `launchSettings.json` settings to test `docfx download` command from Visual Studio.